egs/wsj/s5/steps/durmod/durmodel_train.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2015 Hossein Hadian
+
+num_epochs=100
```

Comment:
Actually this is one of my questions. On a small training set I tried to reach convergence which took ~100 epochs and reached objective value of -0.5 (from starting point -2.9). What type of termination would you suggest? Fixed number of epochs or termination based on objective?